infrastructure/cloud/environments/dev/webapp.tf

```diff
@@ -255,30 +266,130 @@ module "ecs_api_td" {
 
 # Create Web ECS Service
 module "ecs_web_service" {
-  source         = "../../modules/ECS/Service"
-  environment    = var.environment
-  app_name       = var.app_name
-  name           = "web"
-  ecs_cluster_id = module.ecs_cluster.ecs_cluster.id
-  ecs_td_arn     = module.ecs_web_td.ecs_td_arn
-  tg_arn         = module.tg_web.tg_arn
-  sg_id          = data.aws_security_group.app_sg.id
-  subnet_ids     = module.subnets.web_subnets_ids
-  port           = module.ecs_web_td.port
+  source           = "../../modules/ECS/Service"
+  environment      = var.environment
+  app_name         = var.app_name
+  name             = "web"
+  ecs_cluster_id   = module.ecs_cluster.ecs_cluster.id
+  ecs_td_arn       = module.ecs_web_td.ecs_td_arn
+  tg_arn           = module.tg_web.tg_arn
+  sg_id            = data.aws_security_group.app_sg.id
+  subnet_ids       = module.subnets.web_subnets_ids
+  port             = module.ecs_web_td.port
+  ecs_cluster_name = module.ecs_cluster.ecs_cluster.name
+  max_capacity     = 1
+}
+
+# Create CloudWatch Alarms for Web ECS Service
+module "ecs_web_alarms" {
+  source       = "../../modules/Cloudwatch/Alarms"
+  environment  = var.environment
+  app_name     = var.app_name
+  namespace    = "AWS/ECS"
+  service_name = "web-ecs-service"
+  dimensions = {
+    ClusterName = module.ecs_cluster.ecs_cluster.name
+    ServiceName = module.ecs_web_service.service_name
+  }
+  alarm_configurations = [
+    {
+      name                = "cpu-high"
+      metric_name         = "CPUUtilization"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = 70
+      evaluation_periods  = 3
+      period              = 300
+      statistic           = "Average"
+      description         = "CPU utilization sustained above 70% for 15 minutes"
+      alarm_actions       = [module.sns_ecs_alerts.sns_topic_arn]
+    },
+    {
+      name                = "memory-high"
+      metric_name         = "MemoryUtilization"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = 80
+      evaluation_periods  = 2
+      period              = 300
+      statistic           = "Average"
+      description         = "Memory utilization above 80% for 10 minutes"
+      alarm_actions       = [module.sns_ecs_alerts.sns_topic_arn]
+    },
```

Comment:
There should be another alarm for tracking Task count metric. If task count ever goes 0 or less than a defined threshold then we will get notified based on that